docs/docsite/rst/playbooks_delegation.rst

```diff
@@ -252,6 +252,9 @@ use the default remote connection type::
     - hosts: 127.0.0.1
       connection: local
 
+Be aware that changing the connection is not a 'full delegation' so other settings like ``ansible_python_interpreter`` might also need to be overriden.
+Most of the time it is better to use ``local_action`` or ``delegate_to: localhost``.
```

Comment:
Could you add to *"`local_action` or `delegate_to: localhost`"* somehting like *(which are both doing the exact same thing)* so that it is more clear to users ? And still phrase it in such a way that it is unambiguous so people don't think we mean it is the same as `connection: local`...